src/Model/GeneratorParamsList.cs

```diff
@@ -23,48 +23,51 @@
 // </summary>
 // --------------------------------------------------------------------------------------------------------------------
 
-using System; 
+//
+// This source code was auto-generated by AsposeBarcodeCloudCodegen.
+//
 
-namespace Aspose.BarCode.Cloud.Sdk.Model 
+using System;
+
+namespace Aspose.BarCode.Cloud.Sdk.Model
 {
-  using System.Collections;
-  using System.Collections.Generic;
-  using System.Runtime.Serialization;
-  using System.Text;
+    using System.Collections.Generic;
+    using System.Text;
 
-  /// <summary>
-  /// Represents list of barcode builders
-  /// </summary>  
-  public class BarCodeBuildersList 
-  {                       
+    /// <summary>
+    ///     Represents list of barcode generators             
+    /// </summary>
+    public class GeneratorParamsList
+    {
         /// <summary>
-        /// List of barcode builders
-        /// </summary>  
-        public List<BarCodeBuilder> BarCodeBuilders { get; set; }
-		
+        ///     List of barcode generators             
+        /// </summary>
+        public List<GeneratorParams> BarcodeBuilders { get; set; }
```

Comment:
Shouldn't we remove all the mentions of Builder from the project?

Reply:
> Shouldn't we remove all the mentions of Builder from the project?

I suppose we keep it in order of backward compatibility.